README.md

```diff
@@ -11,58 +11,21 @@ Use of NVIDIA cuOpt is subject to the [End User License Agreement](https://docs.
 
 ## [NVIDIA cuOpt](https://developer.nvidia.com/cuopt-logistics-optimization)
 
-cuOpt uses highly optimized GPU-accelerated solvers relying on heuristics, metaheuristics, and optimization. In addition to providing dramatically accelerated, world class solutions to some of the most difficult optimization problems, NVIDIA cuOpt prioritizes ease of use through high level Python and REST based microservice APIs
+cuOpt uses highly optimized GPU-accelerated solvers relying on heuristics, metaheuristics, and optimization. In addition to providing dramatically accelerated, world class solutions to some of the most difficult optimization problems, NVIDIA cuOpt prioritizes ease of use through high level Python SDK and service APIs
```

Comment:
If we remove SDK related things, should we also remove notebooks on them ?

Reply:
I think we will keep a path to the SDK so it will be nice to keep the notebooks too.